Doc/library/socket.rst

```diff
@@ -1365,7 +1365,7 @@ to sockets.
 
    .. availability:: most Unix platforms, possibly others.
 
-   .. versionadded:: 3.3
+   .. versionadded:: 3.8
```

Comment:
This versionadded note is for `recvmsg` - `recv_fds` should get a separate documentation block now that it is its own function.